src/main/java/de/retest/recheck/cli/RecheckCli.java

```diff
@@ -14,6 +14,10 @@
 		subcommands = { Version.class, Diff.class, Commit.class, Ignore.class, Completion.class, CommandLine.HelpCommand.class } )
 public class RecheckCli implements Runnable {
 
+	public static final String REPORT_FILE_PARAM_DESC =
+			"Path to a test report file." + " If the test report is not in the project directory, please specify the"
```

Comment:
The `+` in between is a bit confusing. I would either merge both strings or include a line break in the source code.